features/encrypt/core/src/main/java/org/apache/shardingsphere/encrypt/algorithm/encrypt/RC4EncryptAlgorithm.java

```diff
@@ -30,27 +31,26 @@
  * RC4 encrypt algorithm.
  */
 public final class RC4EncryptAlgorithm implements StandardEncryptAlgorithm<Object, String> {
-    
+
     private static final String RC4_KEY = "rc4-key-value";
-    
+
     private static final int SBOX_LENGTH = 256;
-    
+
     private static final int KEY_MIN_LENGTH = 5;
-    
+
     private volatile byte[] key = new byte[SBOX_LENGTH - 1];
-    
+
     private volatile int[] sBox = new int[SBOX_LENGTH];
-    
+
     @Override
     public void init(final Properties props) {
         reset();
         setKey(props.getProperty(RC4_KEY, "").getBytes(StandardCharsets.UTF_8));
     }
-    
+
```

Comment:
Please keep indentation consistent with the previous line, even empty lines

Reply:
OK, I'll fix it